substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/flow/MethodTypeFlow.java

```diff
@@ -100,7 +101,19 @@ private synchronized void createFlowsGraph(PointsToAnalysis bb, InvokeTypeFlow r
 
                 initFlowsGraph(bb);
             } catch (Throwable t) {
-                /* Wrap all other errors as parsing errors. */
+                /*
+                 * Wrap all other errors as parsing errors, but first print any UnsupportedFeatures
+                 */
+                try {
+                    bb.getUnsupportedFeatures().report(bb);
+                } catch (UnsupportedFeatureException uee) {
+                    System.err.println(uee.getMessage());
```

Comment:
Is there a logger I could use instead of err?

Reply:
Why do you need your own handler here anyway? Wouldn't a surrounding `catch (UnsupportedFeatureException)` take care of a thrown `UnsupportedFeatureException`? cc @cstancu

Reply:
The `UnsupportedFeatureException` is thrown by `bb.getUnsupportedFeatures().report(bb)` which I only invoke when an exception is thrown by the bytecode parser. I am not sure I follow how a surrounding catch could take care of it, unless we through the `UnsupportedFeatureException` instead of a `ParsingError` which doesn't seem to me like the right thing to do.